apps/api/src/app/app.module.ts

```diff
@@ -87,14 +89,17 @@ const autoSchemaFile = environment.production
         fieldMiddleware: [maskOutFieldsMiddleware],
       },
       plugins: [
-        responseCachePlugin({
-          shouldReadFromCache: ({ request: { http } }) => {
-            const bypassCacheKey = http?.headers.get('bypass-cache-key')
-            return bypassCacheKey !== process.env.BYPASS_CACHE_KEY
-          },
-        }),
+        // This was causing problems since graphql upgrade, gives us issues like:
+        // Error: overallCachePolicy.policyIfCacheable is not a function
```

Comment:
Sounds like a node_modules resolution/deduplication issue. 

Not sure how important this cache is but would feel better if we fix it :)

Reply:
We tried fixing this, but weren't successful - Any ideas on what this cache actually does?